ts/src/management/tunnelManagementHttpClient.ts

```diff
@@ -672,9 +672,18 @@ export class TunnelManagementHttpClient implements TunnelManagementClient {
         this.raiseReportProgress(TunnelProgress.StartingRequestConfig);
         const config = await this.getAxiosRequestConfig(tunnel, options, accessTokenScopes);
         this.raiseReportProgress(TunnelProgress.StartingSendTunnelRequest);
-        const result = await this.request<TResult>(method, uri, body, config, allowNotFound);
-        this.raiseReportProgress(TunnelProgress.CompletedSendTunnelRequest);
-        return result;
+        try {
+            const result = await this.request<TResult>(method, uri, body, config, allowNotFound);
+            this.raiseReportProgress(TunnelProgress.CompletedSendTunnelRequest);
+            return result;
+        } catch (error) {
+            if (/certificate/i.test((error as AxiosError<any>).message)) {
+                const originalErrorMessage = (error as AxiosError<any>).message;
+                throw new Error("Tunnel service HTTPS certificate is invalid. This may be caused by the use of a " +
+                    "self-signed certificate or a firewall intercepting the connection." + originalErrorMessage + ". ");
```

Comment:
```suggestion
                    "self-signed certificate or a firewall intercepting the connection. " + originalErrorMessage + ".");
```

Small nit on formatting to do with the spaces.
As is currently in the PR, I believe you may end up with an error message such as `...or a firewall intercepting the connection.Axios Error Message. `.

Reply:
Addressed